lefthook.yml

```diff
@@ -0,0 +1,18 @@
+pre-commit:
+  parallel: true
+  commands:
+    frontend-lint:
+      tags: frontend lint
+      root: "frontend/"
+      glob: "*.{js,jsx,ts,tsx}"
+      run: eslint --fix {staged_files}
```

Comment:
Could you use `yarn lint --fix` instead? I believe we can pass the list of files using the `--file` option. You can check the docs by running:
```bash
npx next lint -h
```